lib/json_elixir/process.ex

```diff
@@ -0,0 +1,82 @@
+defmodule JsonElixir.Process do
+  @invalid_input "expected list or bit string got: "
+
+  def to_html!(json)
+      when is_list(json) or
+             is_bitstring(json) do
```

Comment:
In some situations, the formatter keeps existing live breaks that was manually introduced. This is one of those cases. If you find this formatting a bit weird, you can write the clauses in a single like, like:

```elixir
def to_html!(json) when is_list(json) or is_bitstring(json) do
```
The formatter will not touch it again. Unless, of course, you add more stuff and exceed the `line_length`.